test/unit_tests/arkode/CXX_serial/ark_test_slowerror_brusselator.cpp

```diff
@@ -0,0 +1,611 @@
+/*-----------------------------------------------------------------
```

Comment:
It would be nice to consolidate the common code among this and the other two Brusselator tests (possible a few other places too) into a shared file.

Reply:
Perhaps, but based on the type of test (slow error, accumulated fast error, MRI adaptivity) I need to put different operators into the various RHS functions, so although the parameters and equations are shared, I worry that a common header file would have too many different permutations of RHS functions, potentially obfuscating what actually differentiates one test from another.

Reply:
In a follow-on PR (after this release), we should update the tests to leverage these common utilities.